wdl-lint/src/rules/missing_requirements.rs

```diff
@@ -69,6 +69,19 @@ impl Rule for MissingRequirementsRule {
             SyntaxKind::TaskDefinitionNode,
         ])
     }
+
+    fn related_rules(&self) -> &[&'static str] {
+        &[
+            "RuntimeSectionKeys",
+            "DescriptionMissing",
+            "MatchingParameterMeta",
+            "MissingMetas",
+            "MissingOutput",
+            "MissingRuntime",
```

Comment:
This is a bit tricky 🤔 `MissingRuntime` and `MissingRequirements` _are_ very similar, but also mutually exclusive. The requirements section was added in WDL v1.2 and that also deprecated runtime sections. So `MissingRequirements` will only fire for a v1.2 doc, and `MissingRuntime` will not fire for that doc (and vice versa).

I would remove this